base/scheduler/CLL.jl

```diff
@@ -0,0 +1,188 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+# Also see `work-stealing-queue.h` this is a pure Julia re-implementation
+
+# =======
+# Chase and Lev's work-stealing queue, optimized for
+# weak memory models by Le et al.
+#
+# * Chase D., Lev Y. Dynamic Circular Work-Stealing queue
+# * Le N. M. et al. Correct and Efficient Work-Stealing for
+#   Weak Memory Models
+# =======
+
+module CLL
```

Comment:
Did you update this with my bugfixes?

Reply:
I think I did. the x86 specific stuff was part of them but I also didn't test CLL too much (just that it built)